tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_combo_effects_orientation_zoom.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.tests.graphics.orientation_zoom_base import OrientationZoomBase
+
+
+class TestGfxSmokeTestOZComboEffects(OrientationZoomBase):
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+        self.data_layer.connect_to_wifi()
```

Comment:
actually the latter half of the test opens up the browser and scrolls.  but perhaps I should move this line into orientation_zoom_base.py